run_kraken.py

```diff
@@ -94,6 +94,9 @@ def main(cfg):
             config["performance_monitoring"], "enable_alerts", False
         )
         alert_profile = config["performance_monitoring"].get("alert_profile")
+        pre_chaos_alerts = config["performance_monitoring"].get(
```

Comment:
We need to evaluate the rate of increase between start and end time of the chaos scenario instead of pre chaos. Adding the alerts to https://github.com/krkn-chaos/krkn/blob/main/config/alerts_openshift.yaml will automatically get evaluated between start and end time of chaos.